features/google/src/model/importers/activities-importer-html.js

```diff
@@ -12,15 +12,91 @@ class ActivityHtmlParser {
     _extractDate(text) {
         // Ignoring the timestamp for now - we don't need hourly accuracy at the
         // time of writing this.
```

Comment:
nit (for the future): we might don't care about the time, but it might affects the date if it's displayed on different timezone

Reply:
We don't have that problem rn

Reply:
'right now'...
but if we had tests on this and tried to run them locally on different timezones (as already having team members away from UTC time) this would fail already...

Reply:
(anyway it was a future nit)